sdk/storage/azure-storage-file-share/azure/storage/fileshare/_lease.py

```diff
@@ -123,15 +167,16 @@ def change(self, proposed_lease_id, **kwargs):
         """ Changes the lease ID of an active lease. A change must include the current lease ID in x-ms-lease-id and
         a new lease ID in x-ms-proposed-lease-id.
 
-
         :param str proposed_lease_id:
-            Proposed lease ID, in a GUID string format. The File service returns 400
+            Proposed lease ID, in a GUID string format. The File or Share service returns 400
```

Comment:
Maybe rephrase this to say something like "An error will be raised if the proposed lease ID is not in the correct format".
The actual status code is probably not important to a customer.

Reply:
I agree, sounds good :) will add that